scripts/bnsd/start.sh

```diff
@@ -5,7 +5,7 @@ command -v shellcheck > /dev/null && shellcheck "$0"
 # Choose from https://hub.docker.com/r/iov1/tendermint/tags/
 export BNSD_TM_VERSION=v0.31.5
 # Choose from https://hub.docker.com/r/iov1/bnsd/tags/
-export BNSD_VERSION=v0.21.3
+export BNSD_VERSION=latest
```

Comment:
We have tag v0.22.0 now that we can use: https://hub.docker.com/r/iov1/bnsd/tags